src/accelerate/test_utils/testing.py

```diff
@@ -223,6 +223,19 @@ def setUp(self):
                     shutil.rmtree(path)
 
 
+class AccelerateTestCase(unittest.TestCase):
```

Comment:
Open to calling this `AcceleratorStateTestCase` or `AcceleratorTestCase`, just wanted it to be something obvious where users know *oh if I touch AcceleratorState` somehow they should inherit this